assets/dev/scss/frontend/_container.scss

```diff
@@ -3,8 +3,12 @@
 	--border-radius: 0;
 	--display: flex;
 	--flex-direction: column;
+	// Additional `flex-direction` value, which is used to set set a default value for the parent container.
+	--container-flex-direction: initial;
```

Comment:
I didn't quite get it. Why do you need those `--container-` prefixed ones?

Reply:
@StyleShit 

My logic: 
- for boxed width: parent container is always 'flex row' and the widget direction settings are applied to the inner container only
- for full width: the widget direction settings are applied to the parent container

In this way I can do this without the use of prefix classes. If we are allowed to add prefix classes, then we can simplify the scss.

Reply:
@StyleShit I am no longer using it, because I have introduced a prefix class.